packages/components/src/components/text/text.jsx

```diff
@@ -1,27 +1,27 @@
 import classNames from 'classnames';
 import React from 'react';
 
-const Text = ({ children, size, color, align, weight, lineHeight, as, className }) => {
+const Text = ({ children, size, color, align, weight, line_height, as, className, ...props }) => {
     function setStyle() {
         const style = {
-            '--text-size': `var(--text-size-${size || 's'})`,
-            '--text-color': `var(--text-${color || 'general'})`,
-            '--text-lh': `var(--text-lh-${lineHeight || (as === 'p' ? 'm' : 'xs')})`,
-            '--text-weight': `var(--text-weight-${weight || 'normal'})`,
+            '--text-size': `var(--text-size-${props.size || 's'})`,
+            '--text-color': `var(--text-${props.color || 'general'})`,
+            '--text-lh': `var(--text-lh-${props.line_height || 'm'})`,
+            '--text-weight': `var(--text-weight-${props.weight || 'normal'})`,
```

Comment:
(throughout) `props.` won't include the introduced properties, since `size`, `color`, `line_height` and `weight` is extracted in the argument.